.devcontainer/conda/devcontainer.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Conda",
+  "image": "mcr.microsoft.com/vscode/devcontainers/base:0-bullseye",
```

We've deprecated the `vscode` org images in favor of `devcontainers` as we've developed the open Dev Container Specification (more info [here](https://github.com/microsoft/vscode-dev-containers/issues/1762)).

I'd recommend using the image in https://github.com/devcontainers/images/tree/main/src/base-debian, so something like:

```suggestion
  "image": "mcr.microsoft.com/devcontainers/base:bullseye",
```

---

Thank you!